SCons/Tool/__init__.py

```diff
@@ -809,6 +809,8 @@ def tool_list(platform, env):
         'tar', 'zip',
         # File builders (text)
         'textfile',
+        # Package management
+        'vcpkg',
```

Comment:
Seems like this should be a win32 only thing? Or does it work on all platforms?

Reply:
Looks like win, linux, and macos. So it shouldn't be in default list. (BSD's not supported?, hpux, solaris, aix, other evil remnants of the past which are still in use(that's not referring to BSDs, but rather the others.. ;)

Reply:
furthe review looks like your implementation is win32 only, so should be only in that platforms default tool list above. under `other_plat_tools`

Reply:
My implementation is not Win32-only. I've tested on Ubuntu and Windows, and it should work on Mac as well, though I don't have the ability to test Mac, as I don't own one.

So, it there a way to get this into "default" for supported platforms only? Or should I make users explicitly add it to their Environment's "tools"?

Reply:
Yes. add it to `other_plat_tools` for each supported platform. Also I'd add a note which platforms it's supported on in the docs and CHANGES/RELEASE files.

Reply:
Ummm...how do I detect "linux"? It seems like tools such as m4 and rpm are being added simply based on "not win32". Is there a positive test I can do to detect a linux system?

Reply:
> Ummm...how do I detect "linux"? It seems like tools such as m4 and rpm are being added simply based on "not win32". Is there a positive test I can do to detect a linux system?

Eh, that's a little complex. Depending on where you're asking... a simple way is to check `sys.platform` for `linux`.  However, the name of the Platform module is actually `posix`, which is shared with some of the other choices (that originally comes from the value of `os.name`).